test/common/alerts_mechanism.go

```diff
@@ -174,8 +201,16 @@ func checkAlertManager(ctx *TestingContext) error {
 		return fmt.Errorf("failed to exec to alertmanger pod: %w", err)
 	}
 
-	if !strings.Contains(output, fuseAlertName) {
-		return fmt.Errorf("alert not firing in alertmanager")
+	alertsFiringInAlertManager := false
+	for fuseAlertName := range fuseAlertsToTest {
+		if !strings.Contains(output, fuseAlertName) {
+			alertsFiringInAlertManager = true
+			t.Errorf("%s alert not firing in alertmanager", fuseAlertName)
```

Comment:
the variable and log(?) seem to be saying opposing things: "alerts firing" versus "alert not firing"